### PR TITLE
[#201] Rename the CLI wizard

### PR DESCRIPTION
**Pull request by ixahedron**
## Description

Problem: the CLI wizard we provide can be used to set up a baking
instance or just the Tezos node, but we call it `tezos-baking-wizard`.
This name is a bit confusing.

Solution: renamed the wizard to `tezos-setup-wizard`.

## Related issue(s)

Resolves part of #201 

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../../tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

**Review by pasqu4le — APPROVED** [review]

File: docker/package/model.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
     Unit,
 )
 
+letter_version = "a"
```
Comment (pasqu4le) [inline docker/package/model.py:20]: Can you add a comment (or perhaps a clearer name?) just so we clarify what this is for?

I also think we should mention this in our release template now, as we should reset it to `""` whenever we bump the native release version(s)

**Review by pasqu4le — COMMENTED** [review]
One more tiny nitpick, the PR description says;
```
Resolves part of #
```
(without mentioning the issue number :slightly_smiling_face: )

File: docker/package/model.py (hunk 1 above)
Comment (rvem) [reply docker/package/model.py:20]: Perhaps, it also can be moved to a `TezosBakingServicesPackage` class definition

**Review by rvem — APPROVED** [review]
LGTM, have nothing to add:)

File: docker/package/model.py [hunk 2]
```diff
@@ -446,6 +446,13 @@ def gen_license(self, out):
 
 
 class TezosBakingServicesPackage(AbstractPackage):
+
+    # Sometimes we need to update the tezos-baking package inbetween
```
Comment (rvem) [inline docker/package/model.py:450]: :+1:

**Review by pasqu4le — APPROVED** [review]

**Review by rvem — APPROVED** [review]

**Pull request merged**